.github/workflows/publish-pypi.yaml

```diff
@@ -0,0 +1,21 @@
+name: Python package
+on:
```

Comment:
I would add something along these lines to run this workflow when we push on master.
```
on:
  push:
    branches:
      - master
```